src/main/java/mekanism/client/gui/GuiDictionary.java

```diff
@@ -23,7 +23,7 @@
 @SideOnly(Side.CLIENT)
 public class GuiDictionary extends GuiMekanism
 {
-	public ItemStack itemType;
+	public ItemStack itemType = ItemStack.EMPTY;
```

Comment:
Found a bug while testing, shift right click to open the Dictionaries GUI resulted in a NPE.